.gitignore

```diff
@@ -28,14 +28,16 @@ share/python-wheels/
 *.egg
 MANIFEST
 
-# Files creates when testing the demo project
+# Files created when testing the demo project
 .snakemake/
 /demo/results/
 /demo/reference
 /demo/scratch
 /demo/data/*
 !/demo/data/samples.tsv
 
+demo/run_clc.sh
+
```

Comment:
Maybe we should come up with a more general pattern for excluding files from Git. Maybe we just ignore any files containing `tmp.`. And then, you can rename `demo/run_clc.sh` to either `demo/run_clc.tmp.sh` or `demo/tmp.run_clc.sh` (whatever you prefer). 

If you agree with this approach, then you can replace `demo/run_clc.sh` with `*tmp.*` in the `.gitignore` file.

Reply:
ya I think that's a better approach